db/migrate/20150112083118_create_atmosphere_action_logs.rb

```diff
@@ -0,0 +1,10 @@
+class CreateAtmosphereActionLogs < ActiveRecord::Migration
+  def change
+    create_table :atmosphere_action_logs do |t|
+      t.string :message
+      t.string :log_level
+      t.references :action, index: true
+      t.timestamps
```

Comment:
@nowakowski good catch. What is more `ActionLog` should be immutable. You can [read here](http://stackoverflow.com/questions/3342806/turn-off-updated-at-column-in-rails) how to make the object readonly (but unlike presented example we want `ActionLog` to be remmovable).